cypress/e2e/Counter.cy.ts

```diff
@@ -0,0 +1,21 @@
+describe('Counter', () => {
+    const getCounter = () => cy.get("p").should("have.class", "counter__count");
+    const getIncrementButton = () => cy.get("button").contains('increment');
+    const getDecrementButton = () => cy.get("button").contains('decrement');
+
+    beforeEach(() => {
+        cy.visit("/");
+    })
```

Comment:
the file name should be in lowercase

Reply:
File names renamed, thank you for you comment.